MekHQ/src/mekhq/campaign/mission/ScenarioForceTemplate.java

```diff
@@ -298,6 +304,42 @@ public ScenarioForceTemplate(int forceAlignment, int generationMethod, double fo
         this.deploymentZones = deploymentZones == null ? new ArrayList<>() : new ArrayList<>(deploymentZones);
         this.objectiveLinkedForces = new ArrayList<>();
     }
+    
+    public ScenarioForceTemplate(ScenarioForceTemplate forceDefinition) {
+        forceAlignment = forceDefinition.forceAlignment;
+        generationMethod = forceDefinition.generationMethod;
+        forceMultiplier = forceDefinition.forceMultiplier;
+        deploymentZones = new ArrayList<>();
+        for(int zone : forceDefinition.deploymentZones) {
```

Comment:
```suggestion
        for (int zone : forceDefinition.deploymentZones) {
```

Reply:
These are a code accessibility issue on my end... the words blur together into forint, which is the Hungarian currency...